exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/traces/OtlpStdoutSpanExporterComponentProvider.java

```diff
@@ -26,7 +26,7 @@ public Class<SpanExporter> getType() {
 
   @Override
   public String getName() {
-    return "experimental-otlp/stdout";
+    return "otlp_file/development";
```

Comment:
Is it ok/intentional that all 3 signals have component providers with exactly the same name?

Reply:
Yes! Component providers are resolved based on the combination of `Class<?> getType()` and `String getName()`. We want users to be able to refer to certain concepts which repeat across signals using the same language, like OTLP exporters. In practice, this would look like:
```
logger_provider:
  processors:
    - batch:
         exporter:
           otlp_file/development:

tracer_provider:
  processors:
    - batch:
         exporter:
           otlp_file/development:
```

`otlp_file/development` repeats in both `logger_provider` and `tracer_provider`, and is what instructs this particular component provider implementation to be invoked.

Reply:
OH! Thanks for the clarification, this makes total sense now. 👍🏻